.vsts-ci.yml

```diff
@@ -306,6 +309,16 @@ extends:
             oneESCompat:
               publishTaskPrefix: 1ES.
 
+      ############### SOURCE INDEX ###############
+      - ${{ if eq(variables.enableSourceIndex, 'true') }}:
+        - template: /eng/common/templates-official/job/source-index-stage1.yml@self
+          parameters:
+            sourceIndexBuildCommand: powershell -NoLogo -NoProfile -ExecutionPolicy Bypass -Command "eng/common/build.ps1 -restore -build -ci -nativeToolsOnMachine -configuration $(buildConfiguration)"
+            binlogPath: artifacts/log/$(buildConfiguration)/Build.binlog
+            pool:
+              name: $(DncEngInternalBuildPool)
+              demands: ImageOverride -equals 1es-windows-2022
```

Comment:
I haven't looked at the `source-index-stage1.yml` template, but I'd be surprised if this format for pool information works, since this template is in `templates-official`, which should require the 1ES format for pool information. Meaning, `name`, `image`, and `os`.

Reply:
I've opened an Arcade PR to fix the default pool parameters, and will generally try to wind down the passing of non-default pools within individual repos.

Reply:
https://github.com/dotnet/sdk/pull/42300